gnmvidispine/vs_item.py

```diff
@@ -138,7 +139,7 @@ def fromXML(self, xmldata=None, objectClass="item"):
         :param objectClass: is this an item or collection
         :return: self
         """
-        if isinstance(xmldata,basestring):
+        if isinstance(xmldata,str):
```

Comment:
is this ok in py2 without future aliases loaded?

Reply:
Hard to tell as large parts of the file break when you remove the future aliases import.